src/sw.ts

```diff
@@ -184,6 +184,11 @@ async function requestRouting (event: FetchEvent, url: URL): Promise<boolean> {
   } else if (!isValidRequestForSW(event)) {
     trace('helia-sw: not a valid request for helia-sw, ignoring ', event.request.url)
     return false
+  } else if (url.href.includes('bafkqaaa.ipfs')) {
+    // TODO: we shouldn't need to do this, but sometimes verified-fetch will attempt to fetch this.
```

Comment:
i was getting failures on the bafkqaaa "no gateways could find this block" but verified that helia and unixfs handles it properly (https://codepen.io/SgtPooki/pen/xxedpGP).

We need to investigate with verified-fetch.

Reply:
checked with verified fetch, and it seems to work fine there too, not sure why it fails in helia-service-worker-gateway, but this check is pretty lightweight and should prevent extra unnecessary network call: https://codepen.io/SgtPooki/pen/xxedpGP

will update comment